css/main.css

```diff
@@ -146,6 +146,11 @@ html, body {
     margin: 0;
 }
 
+.leaflet-popup-content a {
+    display: block;
```

Comment:
Why is `display: block;` needed?

Reply:
I'm using `display: block;` along with `line-height` to increase the touch area for the link

Reply:
anchor tag has `display: inline` by default

Reply:
Ah nice. Thank you for the explanation. Do you think adding the reason as a comment on the attribute is useful for others?

Reply:
Sure. I'll add a comment